{{cookiecutter.repo_name}}/.coveragerc

```diff
@@ -0,0 +1,10 @@
+[run]
+data_file = .coverage
+omit = {{cookiecutter.repo_name}}/settings*
+    {{cookiecutter.repo_name}}/conf*
+    *wsgi.py
+    *migrations*
+    *admin.py
+    *test*
```

Comment:
What's the benefit? Shouldn't the value always be 100%, or will this measurement identify test code that is not run?